lib/ui/painting/flutter_rtree.h

```diff
@@ -0,0 +1,103 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_LIB_UI_PAINTING_FLUTTER_RTREE_H_
+#define FLUTTER_LIB_UI_PAINTING_FLUTTER_RTREE_H_
+
+#include "third_party/skia/include/core/SkBBHFactory.h"
+#include "third_party/skia/include/core/SkTypes.h"
+
+namespace flutter {
+/**
```

Comment:
Please follow the Doxygen documentation convention [followed elsewhere](https://github.com/flutter/engine/blob/482155b3c6ed17e59129655a57936c61eb11b8c2/shell/common/shell.h#L97) in the engine.